### PR TITLE
Rubber chicken toy

### PR DESCRIPTION
**Pull request by xprospero**
## About the PR
Added a rubber chicken toy.

## Why / Balance
It's fun!

## Technical details
The rubber chicken toy is basically just a plushie.

## Media

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/space-wizards/space-station-14/assets/116504990/5041f5ee-f0f1-4912-9693-bc3893d31e5d

## Breaking changes

**Changelog**

None

- add: Rubber chicken toy!

### CODE REVIEW

**Comment by lzk228** [comment]
is it copyrighted?

**Comment by Hrosts** [comment]
I think for some of the plushie additions it was requested to remove the default plushie squeak it makes together with its own sound when thrown.

**Comment by xprospero** [comment]
> is it copyrighted?

No, I make-a all the assets.

**Comment by xprospero** [comment]
> I think for some of the plushie additions it was requested to remove the default plushie squeak it makes together with its own sound when thrown.

good idea, I just fixed it

**Comment by lzk228** [comment]
> > is it copyrighted?
> 
> No, I make-a all the assets.

i mean not the sprites itself but the idea how its looks
like i remember #15018 was closed because crocs are licensed

**Comment by xprospero** [comment]
> > > is it copyrighted?
> 
> > 
> 
> > No, I make-a all the assets.
> 
> 
> 
> i mean not the sprites itself but the idea how its looks
> 
> like i remember #15018 was closed because crocs are licensed

good catch.

i've done some searching and as far as I am aware the idea itself doesn't have a patent or copyright attached to it, neither does the design the sprite uses.

the origin of the rubber chicken itself is obscure and there are lots of different manufacturers online offering what seems to be an identical product.

**Comment by xprospero** [comment]
> Are you still working on this? I would love to see it in the game. If you fix the merge conflict I will bring it up for maintainer vote. The only hurdle would be to find out if this has any copyright or not.

The assets are made by me and the concept has no copyright attached to it.

**Comment by slarticodefast** [comment]
I think there is currently no method to obtain it in-game. Could you add one to the toy crate that can be ordered at cargo?

**Comment by slarticodefast** [comment]
> also to maints closet :trollface:

I approve, but it should be rare enough.

**Comment by UbaserB** [comment]
You coming back to this?

**Comment by xprospero** [comment]
> You coming back to this?

I intend to but I've been quite busy recently 😿 

I'll try to do this tomorrow lol

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -1546,3 +1546,51 @@
...
+      params:
+        variation: 0.125
+  - type: Food
+    requiresSpecialDigestion: true
+    useSound:
+      collection: RubberChicken
+      params:
+        variation: 0.125
+  - type: MeleeWeapon
+    soundHit:
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:1590]:
```suggestion
  - type: MeleeWeapon
    wideAnimationRotation: 110
    soundHit:
```

I went ahead and fixed the wideswing animation rotation for you. I will merge as soon as the tests are done :)

**Comment by slarticodefast** [comment]
Thank you for your contribution!

**Pull request merged**